opencti-platform/opencti-front/src/private/Root.tsx

```diff
@@ -1,6 +1,9 @@
 import { StyledEngineProvider } from '@mui/material/styles';
-import React, { FunctionComponent } from 'react';
+import React, { FunctionComponent, useEffect, useState } from 'react';
 import { graphql, usePreloadedQuery, PreloadedQuery } from 'react-relay';
+import { AnalyticsProvider } from 'use-analytics';
+import googleAnalytics from '@analytics/google-analytics';
+import Analytics from 'analytics';
```

Comment:
analytics lib works with a plugin system so if needed we can later add more analytiucs providers